### PR TITLE
Adding ability to build without deps

### PR DESCRIPTION
**Pull request by t-etfali**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -5,7 +5,7 @@ GOGET=${GOCMD} get
 GOTEST=${GOCMD} test
 GORUN=${GOCMD} run 
 
-BINARY_NAME=bin/guest-agent-test-extension
+BINARY_NAME=bin/GuestAgentTestExtension
```
Comment (larohra) [inline Makefile:8]: why did you change the name? Any particular reason?

Comment (t-etfali) [reply Makefile:8]: None of the other extensions had this format for their name so I wanted to match that

Comment (larohra) [reply Makefile:8]: Ahh ok makes sense

**Review by larohra — APPROVED** [review]
